site/content/3.12/aql/high-level-operations/upsert.md

```diff
@@ -63,6 +66,49 @@ UPDATE { logins: OLD.logins + 1 } IN users
 Note that in the `UPDATE` case it is possible to refer to the previous version of the
 document using the `OLD` pseudo-value.
 
+## Filter matching syntax (experimental)
+
+The alternative syntax for `UPSERT` operations allows you to use dynamic
+attribute names to look up documents. This means that the expression you use to
```

Comment:
I would not call it "dynamic attribute names". I would rather call it "more flexible filter conditions".